fnet/cli/predict.py

```diff
@@ -213,7 +217,7 @@ def save_args_as_json(path_save_dir: str, args: argparse.Namespace) -> None:
         )
     with open(path_json, 'w') as fo:
         json.dump(vars(args), fo, indent=4, sort_keys=True)
-        print('Saved:', path_json)
+        logger.info('Saved: {path_json}')
```

Comment:
Why is the logger statement indented into the open statement?

Reply:
Originally, the intent was to only log a message if a json was actually written. Now that I think about it though, if a json was not written because of, say, and IO error, then there would be some exception and the program would crash out anyway. Anyway, I'll fix this. :)

Reply:
I added another commit to pull the logging statement outside of the context block.